apps/pie-microsite/src/_11ty/shortcodes/contentPageImage.js

```diff
@@ -17,20 +17,21 @@ const createCaption = config => (config.caption
 module.exports = function (config) {
     const context = config.context ?? 'contentPage';
     const contextClass = `c-${context}-img`;
+    const figureClasses = [contextClass, 'c-contentImage'];
+    const isSpreadImage = config.type === 'spread';
+    const imageStyles = !isSpreadImage ? `style="--img-width: ${config.width};"` : '';
```

Comment:
Not sure if this has been resolved? (can't see anything added to the function comments)

Reply:
Sorry, I think I must've clicked resolved too fast 😬 
I've updated it now so that if a width isn't provided the image will fill the entire container and changed `isSpreadImage` to `isImageFullContainerWidth`.
Hopefully that's clearer? But happy to change it

Reply:
I think that's way clearer now 👍